server/src/com/cloud/network/vpn/RemoteAccessVpnManagerImpl.java

```diff
@@ -596,7 +597,7 @@ public void doInTransactionWithoutResult(TransactionStatus status) {
 
 
         sb.and("id", sb.entity().getId(), SearchCriteria.Op.EQ);
-        sb.and("username", sb.entity().getUsername(), SearchCriteria.Op.EQ);
+        sb.and("username", sb.entity().getUsername(), SearchCriteria.Op.LIKE);
```

Comment:
@ustcweizhou : Thanks but unfortunately, addOr & addAnd methods are now deprecated, so I avoided using the same.

Also, LIKE with work only when keyword is set. When only username is set,      `sc.setParameters("username", username); `
will get executed with the exact match i.e. same as how EQ will work!